tethys_cli/proxyapps_commands.py

```diff
@@ -4,188 +4,167 @@
     write_info,
 )
 
-from pathlib import Path
+from tethys_cli.cli_helpers import setup_django
 
-from sqlalchemy import create_engine, exc
-from sqlalchemy.ext.automap import automap_base
-from sqlalchemy.orm import sessionmaker
-from tethys_apps.utilities import (
-    get_tethys_home_dir,
-)
-from .settings_commands import read_settings
-
-TETHYS_HOME = Path(get_tethys_home_dir())
+from django.db import IntegrityError
 
 
 def add_proxyapps_parser(subparsers):
     # Setup list command
     proxyapps_parser = subparsers.add_parser(
         "proxyapp", help="Add proxy apps and list proxy apps into the Tethys Platform"
     )
-    proxyapps_parser.add_argument(
-        "-l",
-        "--list",
-        help="list available proxy apps in the current tethys installation",
-        action="store_true",
+    proxyapps_subparsers = proxyapps_parser.add_subparsers(title="Options")
+
+    proxyapps_list_parser = proxyapps_subparsers.add_parser(
+        "list", help="list available proxy apps in the current tethys installation"
     )
-    proxyapps_parser.add_argument(
-        "-a",
-        "--add",
+    proxyapps_list_parser.set_defaults(func=list_proxyapps)
+
+    proxyapps_add_parser = proxyapps_subparsers.add_parser(
+        "add",
         help="Add a new proxy app. Arguments: proxy_app_name endpoint [description] [logo_url] [tags] [enabled] [show_in_apps_library] [back_url] [open_new_tab] [display_external_icon] [app_order]",
-        nargs="+",
     )
-    proxyapps_parser.add_argument(
-        "-u",
-        "--update",
-        help="Update a new proxy app. Arguments: proxy_app_name property new_value ",
-        nargs="+",
+    proxyapps_add_parser.add_argument(
+        "proxy_app_name",
+        help="The proxy app name",
```

Comment:
Add examples to all `help` descriptions like so:

```suggestion
        help='The proxy app name (e.g.: "My App").',
```

Reply:
Done

Reply:
@romer8 can you add "(e.g. ...)" examples to all the help text?